docs/about/release-notes.rst

```diff
@@ -3,29 +3,48 @@
 Release Notes
 =============
 
-Since v0.4
+Since v0.5
 ----------
 
 Features
 ~~~~~~~~
 
-* New ``profiler`` plotting functionality where one of the slider dimensions can be displayed as a profile in a subplot (1D and 2D projections only).
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+Contributors
+~~~~~~~~~~~~
+
+v0.5 (January 2021)
+-------------------
+
+Features
+~~~~~~~~
+
+* New ``profile`` plotting functionality where one of the slider dimensions can be displayed as a profile in a subplot (1-D and 2-D projections only).
```

Comment:
Could make this briefer and just link to the relevant docs https://scipp.github.io/visualization/plotting-nd-data.html#Profile-picking

Reply:
See below.